packages/block-library/src/social-link/icons/x.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+export const XIcon = () => (
+	<SVG width="24" height="24" viewBox="0 0 24 24" version="1.1">
```

Comment:
The logo is slightly off-center. Using the logo from the [X brand guidelines](https://about.twitter.com/en/who-we-are/brand-toolkit), this path should work slightly better:

```
<Path d="M13.982 10.622 20.54 3h-1.554l-5.693 6.618L8.745 3H3.5l6.876 10.007L3.5 21h1.554l6.012-6.989L15.868 21h5.245l-7.131-10.378Zm-2.128 2.474-.697-.997-5.543-7.93H8l4.474 6.4.697.996 5.815 8.318h-2.387l-4.745-6.787Z" />
```

Reply:
👍 Done in f4154f040afe50e6b7e3296de49524c188ae59a2